src/resources/views/buttons/delete.blade.php

```diff
@@ -1,52 +1,55 @@
 @if ($crud->hasAccess('delete'))
-	<a href="{{ url($crud->route.'/'.$entry->getKey()) }}" class="btn btn-xs btn-default" data-button-type="delete"><i class="fa fa-trash"></i> {{ trans('backpack::crud.delete') }}</a>
+	<a href="javascript:void(0)" onclick="deleteEntry(this)" data-route="{{ url($crud->route.'/'.$entry->getKey()) }}" class="btn btn-xs btn-default" data-button-type="delete"><i class="fa fa-trash"></i> {{ trans('backpack::crud.delete') }}</a>
```

Comment:
That works and I'm not sure if I like it.

If we're going to be prevent the default action, we should be preventing the default action (and not making peple wonder why there's a `javascript:void(0)` instead).